presets.go

```diff
@@ -151,9 +151,12 @@ type VideoTarget struct {
 	Profile           string         `json:"profile,omitempty"`
 	Level             string         `json:"level,omitempty"`
 	Tune              string         `json:"tune,omitempty"`
-	MinGOPFrames      int            `json:"min_gop_frames,omitempty"`
-	MaxGOPFrames      int            `json:"max_gop_frames,omitempty"`
-	ExactGOPFrames    int            `json:"exact_gop_frames,omitempty"`
+	MinGOPFrames      *int           `json:"min_gop_frames,omitempty"`
```

Comment:
These look good except for the data type, curious why you chose int pointers here?

Reply:
I thought they needed to be pointers to remove the fields/values completely from the payload. Or can the data type be int, and the provided value when initiating the struct be a reference?

Reply:
Ahh yeah, that makes sense for complex structures that do not support the `omitempty` tag. Thankfully `int` does, so any values that would be `0` that have the `omitempty` will be omitted from the json when marshaling.

Reply:
good to know its for complex structs only! I updated.

Reply:
Yeah or any time where `0` (or the "empty") value should be considered a valid value. For instance, if the prop were `volume` then `0` could be a valid value and we'd not want to `omitempty` and could use a pointer.